code/style.css

```diff
@@ -1,7 +1,758 @@
+}
+
+/* Introduction */
+.intro {
+  padding: 0 15px;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 500px;
+}
+
+.intro__text {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.25;
+  color: #2D2E2F;
+}
+
+/* FAQ Accordion Section */
+.faq {
+  background-color: #2D2E2F;
+  border-radius: 5px;
+  overflow: hidden;
+  max-width: 500px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.faq__title {
+  padding: 15px 0 20px;
+  color: white;
+}
+
+.faq__text {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 1.5;
+  padding: 10px 0;
+}
+
+.faq__text--center {
+  text-align: center;
+}
+
+/* Styling the Accordion Button for Closing/Opening */
+  .faq__question {
+    text-align: left;
+    font-size: 18px;
+    width: 100%;
+    border: none;
+    outline: none;
+    transition: 0.4s;
+    padding: 10px 25px;
+    color: #2d2e2f;
+    background-color: #DCD8DC;
+  }
+
+/* Adding background color to button when clicked on and when moused over */
+/* Add .active in JS */
+  .faq__question--active, .faq__question:hover {
+    background-color: #726a6a;
+    color: #DCD8DC;
+    cursor: pointer;
+  }
+
+/* Styling the accordion panel / answer (hidden by default */
+  .faq__answer {
+    display: none;
+    padding: 0 25px;
+    color: #2d2e2f;
+    background-color: #bfb8b9;
+  }
+
+/* Adding + / - symbols to the FAQ question */
+
+/* Plus / Minus Option Unicode - I prefer Emojis, Commented Out */
+/* .faq__question:after {
+  content: "\271A";
+  float: right;
+}
+
+.active:after {
+  content: "\2212";
+  color: #DCD8DC;
+} */
+
+/* Emoji Option */
+.faq__question:after {
+  content: "\1F607";
+  float: right;
+}
+
+.faq__question--active:after {
+  content: "\1F608";
+  color: #DCD8DC;
+}
+
+
+/* Sign-up Form */
+.signup {
+  background-color: #2D2E2F;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 500px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.signup__form {
+  display: flex;
+  flex-direction: column;
+}
+
+/* Sign-up Form Title */
+.signup__image {
+  height: 60px;
+  width: 60px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.signup__title {
+  color: white;
+  padding: 10px 0;
+}
+
+/* Sign-up Form Input Fields */
+
+/* Placeholder Text Styling */
+::placeholder {
+  color: #2D2E2F;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+}
+
+/* Input Fields Styling */
+input[type=text], input[type=tel], input[type=email] {
+  width: 100%;
+  height: 40px;
+  padding: 0 15px;
+  border-radius: 5px;
+  border: none;
+  margin-bottom: 10px;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+}
+
+/* Options Text */
+.signup__options {
+  color: #dcd8dc;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+}
+
+
+/* Radio Buttons */
+/* Remove Styling from Radio Buttons */
+.radio--remove {
```

Comment:
These parts are just awesome. So much code for that small amount of customization. A gold medal for your big effort on this one.